_posts/2018-10-02-release-of-text2num-1.md

```diff
@@ -10,16 +10,32 @@ tags: python NLP
 
 The output of speech-to-text systems are entirely made of words, without punctuation or capitalization. This makes visual scanning for numbers quite cumbersome,
 especially in transcriptions of real life dialogues as they also contain a lot of gibberish words — like « heu, ben, bah… » — and the syntax or
-grammar is not always right. Besides that, text mining tools and techniques often, if not always, expect numbers to be in decimal digit representation.
+grammar is not always correct. Besides that, text mining tools and techniques often, if not always, expect numbers to be in decimal digit representation.
 
 That's why we decided to add a transformation pass to our speech-to-text engine in order to convert all spoken numbers into their digit spelling.
 
-Here at Allo-Media, we are fond of Open Source Software. So we first looked at the state of the art of [Python](http://www.python.org) libraries for parsing words into numbers. There were some for the English language, but we didn't find any for French. Therefore, we decided to build our own library and contribute it back to the community.
+Here at Allo-Media, we are fond of Open Source Software. So we first looked at the state of the art of [Python](http://www.python.org) libraries for parsing words into numbers. There were at least [one](https://pypi.org/project/word2number/) for the English language, but we didn't find any for French. Therefore, we decided to build our own library and contribute it back to the community.
+
+We could have ported the [Word2number](https://pypi.org/project/word2number/) library, but it has some flaws:
+
+- It is unable to detect by itself the bounds of a number expression;
+- its algorithm is weak (ex: `w2n.word_to_num('hundred five fifty') == 550`);
+- French has some pecularities like « *quatre-vingt-dix-neuf* » vs « *nonante-neuf* ».
+
+So we started a linguistic parser from scratch that is able to identify numbers and correctly isolate contiguous ones in a sequence. Moreover, we wanted it to be able
+to parse different flavors of french (e.g. *soixante-dix* and *septante* for 70, etc…).
+
+If you are interested in linguistics, *septante* for 70 and *nonante* for 90 are used in Belgium, Switzerland, Luxembourg, Aosta Valley, Jersey French and to a lesser extend in French regions of Savoie, Franche-Compté and even sometimes in Lorraine and Provence (source [Wikipedia](https://fr.wikipedia.org/wiki/70_(nombre)#Linguistique)). The rest of the French speaking world uses respectively *soixante-dix* and *quatre-vingt-dix*. The usage of *Huitante* and *octante* instead of *quatre-vingts* is [more restricted yet](https://fr.wikipedia.org/wiki/80_(nombre)#Huitante).
```

Comment:
Nit: *more restricted yet*: just `more restricted` sounds slightly better and idiomatic to me.